driver/src/main/com/mongodb/reactivestreams/client/internal/MongoClientImpl.java

```diff
@@ -73,4 +82,24 @@ public MongoClientSettings getSettings() {
     public <TResult> ListDatabasesPublisher<TResult> listDatabases(final Class<TResult> clazz) {
         return new ListDatabasesPublisherImpl<TResult>(wrapped.listDatabases(clazz));
     }
+
+    @Override
+    public ListDatabasesPublisher<Document> listDatabases(final ClientSession clientSession) {
+        return listDatabases(clientSession, Document.class);
+    }
+
+    @Override
+    public <TResult> ListDatabasesPublisher<TResult> listDatabases(final ClientSession clientSession, final Class<TResult> clazz) {
+        return new ListDatabasesPublisherImpl<TResult>(wrapped.listDatabases(clientSession, clazz));
+    }
+
+    @Override
+    public Publisher<ClientSession> startSession(final ClientSessionOptions options) {
```

Comment:
Add a test for this.

Reply:
Good catch. Done.

Reply:
IntelliJ caught it with its unused method warning.